hera_qm/ant_metrics.py

```diff
@@ -475,100 +793,118 @@ def iterative_antenna_metrics_and_flagging(self, crossCut=5, deadCut=5, alwaysDe
             else:
                 break
 
-    def save_antenna_metrics(self, metricsJSONFilename):
-        '''Saves all cut decisions and meta-metrics in a human-readable JSON that can be loaded
-        back into a dictionary using hera_qm.ant_metrics.load_antenna_metrics().'''
+    def save_antenna_metrics(self, filename):
+        """Output all meta-metrics and cut decisions to HDF5 file.
+
+        Saves all cut decisions and meta-metrics in an HDF5 that can be loaded
+        back into a dictionary using hera_qm.ant_metrics.load_antenna_metrics()
```

Comment:
the ant_metrics.load_antenna_metrics is just a wrapper around metrics_io.load_metric_file. It really doesn't need to exist but it helps with backwards compatibility and M&C compatibility since the M&C setup calls ant_metrics.load_antenna_metrics